### DISCUSSION TITLE
Github Streak stats broken- 23.11.2022

### DISCUSSION THREAD

**Discussion by KatarzynaChmielecka** [DenverCoder1/github-readme-streak-stats#358] [Q&A]
Does anyone also have this problem?
I put 3 commits yesterday and my streak doesn't change... Also, I tried to put an empty commit with yesterday's date and it also doesn't work :(

**Comment by DenverCoder1** [comment]
It does not show on your profie
![image](https://user-images.githubusercontent.com/20955511/203858028-b7d147aa-14a0-4bed-bced-ab39816cdc14.png)

Are they in a private repository?

https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/faq.md#how-do-i-enable-private-contributions

**Reply by KatarzynaChmielecka** [reply]
No, they're not in private repositories, but I have enabled private repos.

**Reply by KatarzynaChmielecka** [reply]
![image](https://user-images.githubusercontent.com/56206231/203859501-0a1e8ed8-ef6b-4c2e-a530-140e102fc757.png)
you can also see the same on 17 Oct, but it didn't broke streak...

**Reply by KatarzynaChmielecka** [reply]
And while this didn't help? git commit --date="2022-11-23 12:00" -m "Test commit" --allow-empty

**Reply by DenverCoder1** [reply]
That should add a commit at 12:00 in your timezone. Did you also push the commit?

**Reply by KatarzynaChmielecka** [reply]
as you can see I add this 3 times... but in pr all of them has the actual date... 
![image](https://user-images.githubusercontent.com/56206231/203865767-53e5935b-004a-4fee-a8f8-9a2449aae919.png)

**Reply by KatarzynaChmielecka** [reply]
Merging pr caused repaired streak :))) I'm happy because this motivates me ;)

**Reply by DenverCoder1** [reply]
GitHub only counts commits on main branches so that's probably why.

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Reply by KatarzynaChmielecka** [reply]
No, it's not that. Earlier commits on pr also were included :)

**Reply by Cfomodz** [reply]
@KatarzynaChmielecka Sounds like this is fixed so can probably mark this as answered :)
